client/electron/vpn_service.ts

```diff
@@ -0,0 +1,63 @@
+// Copyright 2024 The Outline Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {invokeMethod} from './go_plugin';
+import {
+  StartRequestJson,
+  TunnelStatus,
+} from '../src/www/app/outline_server_repository/vpn';
+
+interface VpnConfig {
```

Comment:
Let's call this EstablishVpnRequest since it conflates the id and the config and I don't want to confuse it with the tunnel config.

Reply:
Done.